crates/wdk-build/src/utils.rs

```diff
@@ -98,6 +98,19 @@ pub fn detect_wdk_content_root() -> Option<PathBuf> {
         );
     }
 
+    // If NugetWdkContentRoot is present in environment, use it
+    if let Ok(wdk_content_root) = env::var("NugetWdkContentRoot") {
```

Comment:
I don't think we should be doing custom handling here for nuget since `NugetWdkContentRoot` doesn't exist normally for the Kit. How does building with nuget generally work in a non-rust environment?

Reply:
I understand. The C samples use the [Build-SampleSet.ps1 script](https://github.com/microsoft/Windows-driver-samples/blob/main/Build-SampleSet.ps1). Adding the `NugetWdkContentRoot` was one of the ways to point to the NuGet pkg root. I'm not fully aware of how msbuild works with NuGet pkgs and how SxS WDK installations work.

Reply:
I very much do not want to add new scaffolding to windows-drivers-rs when existing mechanisms exist. In order to make sure this doesn't diverge from the original kit, I think you need to find out how C/C++ WDK handles pointing to the WDKContentRoot when using the new nuget install methods. @NateD-MSFT do you have insight/contacts for this?

Reply:
I'll spin up a thread with the WDK team to ask.